src/scale.ts

```diff
@@ -273,6 +275,7 @@ export function scaleTypeSupportProperty(scaleType: ScaleType, propName: string)
     case 'domain':
     case 'range':
     case 'scheme':
+    case 'interpolate':
```

Comment:
Esp. it's just one line, it's very easy for me to overlook this and we leave bug like this in the codebase.